src/state/items.ts

```diff
@@ -1,30 +1,42 @@
-import * as mintabseAapi from 'mintbase'
 import { MintMetadata } from 'mintbase'
+import { formatNearAmount } from 'near-api-js/lib/utils/format'
 import { selectorFamily } from 'recoil'
 import urlcat from 'urlcat'
 import { IWallet, mintbaseContract, network } from './near'
 
-export interface ItemWithMetadata {
-  id: string
-  minter: string
-  tokens: Token[]
-  thing: MintMetadata
+export interface ThingWithMetadata {
```

Comment:
Let's rename Thing -> Item everywhere

Reply:
Also, I would prefer not to mention 'Metadata', let's have just an Item with data. I guess we wouldn't need it without Mintbase metadata.

Reply:
Yes, this should be an Item